jsk_arc2017_common/node_scripts/publish_dataset_v3.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+
+import os
+import os.path as osp
+
+import numpy as np
+import skimage.io
+import yaml
+
+import cv_bridge
+import dynamic_reconfigure.server
+from geometry_msgs.msg import TransformStamped
+import genpy.message
+import rospy
+from sensor_msgs.msg import CameraInfo
+from sensor_msgs.msg import Image
+from std_msgs.msg import Header
+import tf
+
+from jsk_arc2017_common.cfg import PublishDatasetV3Config
+
+
+class DatasetCollectedOnShelfMultiViewScenes(object):
+
+    def __init__(self):
+        self.scene_ids = []
+        self.root = '/data/projects/arc2017/datasets/JSKV3_scenes'
```

Comment:
The dataset directory should be defined as this? (NOT `/data/...`)
```
import rospkg
PKG_DIR = rospkg.RosPack().get_path('jsk_arc2017_common')
self.root = osp.join(PKG_DIR, 'data/datasets/JSKV3_scenes')
```

Also, please add dataset installation code to `install_data.py` or something like that.